CMakeLists.txt

```diff
@@ -121,6 +122,45 @@ find_package(Threads REQUIRED)
 # Set up directories
 include_directories(${PROJECT_SOURCE_DIR}/include)
 
+# Set up pathes to javascript
+
+if (BENCHMARK_DOWNLOAD_JS)
+    set(JQUERY_NAME "jquery-2.2.1.min.js")
+    set(HIGHSTOCK_NAME "highstock.js")
+    set(HIGHSTOCK_MORE_NAME "highcharts-more.js")
+
+    file(DOWNLOAD "https://code.jquery.com/jquery-2.2.2.min.js" "${PROJECT_BINARY_DIR}/include/benchmark/${JQUERY_NAME}" SHOW_PROGRESS STATUS state)
```

Comment:
[Line 14](https://github.com/LepelTsmok/benchmark/blob/master/CMakeLists.txt#L14) makes the download optional. The default is off, which should be what you want.